keycloak/themes/hmda/login/template.ftl

```diff
@@ -60,6 +60,7 @@
               <div class="usa-alert-body">
                 <h3 class="usa-alert-heading">${message.type} Status</h3>
                 <p class="usa-alert-text">${message.summary}</p>
+                <p>For help account-related issues, please contact <a href="mailto:${properties.supportEmail!}">${properties.supportEmail}</a></em>.</p>
```

Comment:
I'll cleanup the dangling `</em>`.  I'm going to hold off on removing the `mailto` link.  I realize we talked about removing, but I'd like to demo it to the group once first.  If we all agree we don't want it, I'll dump it.